05-EventAggregator/src/PrismSample.Android/PrismSample.Android.csproj

```diff
@@ -54,7 +54,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Prism.DryIoc.Forms" Version="$(PrismVersion)" />
-    <PackageReference Include="Xamarin.Forms" Version="$(XamarinFormsVersion)" />
+    <PackageReference Include="Xamarin.Forms" Version="4.5.0.617" />
```

Comment:
We maintain the version of Xamarin.Forms globally to ensure that the samples stay in sync and are easier to manage